frame/subframe.go

```diff
@@ -383,7 +383,7 @@ func (subframe *Subframe) decodeRicePart(paramSize uint) error {
 		if err != nil {
 			return unexpected(err)
 		}
-		if paramSize == 4 && x == 0xF || paramSize == 4 && x == 0x1F {
+		if paramSize == 4 && x == 0xF || paramSize == 5 && x == 0x1F {
```

Comment:
Haha, sorry. I just read your last paragraph where you mentioned that you don't have any file to test this. Making a mental note to read the entire pull request before sending a reply :)